docs/manual/java/guide/logging/Logging.md

```diff
@@ -25,3 +25,29 @@ If you'd like to use the Lagom logger module on a project that doesn't have the
 @[lagom-logback-libdep](code/build-log.sbt)
 
 The Lagom logger module includes a default logback configuration. Read [[Configuring Logging|SettingsLogger]] for details.
+
+## Log4j 2
+
+Lagom can be configured to use [Log4j 2](https://logging.apache.org/log4j/2.x/) as its default logging engine. Using Log4j 2 can use either the SLF4J API for logging as shown above or the Log4j 2 API. For example, using the Log4j 2 API:
+
+@[](code/docs/logging/Log4j2Example.java)
+
+If you're using maven, you'll need to add the Lagom log4j2 dependency to your classpath to get Log4j 2 support:
+
+```xml
+<dependency>
+    <groupId>com.lightbend.lagom</groupId>
+    <artifactId>lagom-log4j2_2.11</artifactId>
+    <version>${lagom.version}</version>
+</dependency>
+```
+
+If you're using sbt, you'll have to disable the `LagomLogback` plugin and enable the `LagomLog4j2` plugin instead. So for example, if you're using the `LagomJava` plugin:
```

Comment:
FYI, we're going to change the mechanism by which Lagom's logback support is added so that you don't need to disable the `LagomLogback` plugin when adding `LagomLog4j2`, but we'll modify these docs when we do that, so this is fine for now.